runtime/client/CMakeLists.txt

```diff
@@ -264,10 +265,8 @@ else(WIN32)
 	find_package(SDL2 REQUIRED)
 
 	include_directories(../kernel/src/sys/linux
-		../shared/src/sys/linux
-		${SDL2_INCLUDE_DIRS})
```

Comment:
if you're hoisting the include dirs outside the platform specific section, please also bring out the `find_package(SDL2)` above